src/tracing/backend.h

```diff
@@ -63,6 +63,7 @@ class Backend : public ::chip::IntrusiveListNodeBase<>
     /// Trace a zero-sized event
     virtual void TraceInstant(const char * label, const char * group) {}
 
+    virtual void TraceCounter(const char * label, const char * group) {}
```

Comment:
This needs some explanation why a "counter" is needed. If we assume all trace events are captured, a counter seems to be very similar to an instant or anything else that you can count the number of invocations.